src/filePush/filePush.ts

```diff
@@ -0,0 +1,29 @@
+import Client from 'ssh2-sftp-client';
+import logger from '../util/logger';
+
+
+export const filePush = async (filename: string) => {
+
+  const config = {
+    host: process.env.SFTP_Address,
+    username: process.env.SFTP_User,
+    password: process.env.SFTP_Password,
+    retries: 3, 
+  };
+  
+  const sftp = new Client();
+
+  const remoteFolderLocation = '/upload/';
+  const remoteFileLocation = remoteFolderLocation + filename;
+  
+  await sftp.connect(config).then(() => {
```

Comment:
This one is completely optional and an open-ended question. Separating callbacks into individual lines really aids with readability. But I do know that Prettier gets in the way of doing this. For complex calls, I got around this by adding a `//` after the 1st line. So, it would look like this:
```TypeScript
  await sftp //
    .connect(config)
    .then(() => sftp.fastPut(filename, remoteFileLocation))
    .then(() => {
      logger.info('Successfully uploaded to SFTP');
      return sftp.end();
    })
    .catch((err) => {
      logger.error('', err);
      throw err;
    });
```

What are thoughts on this?

Reply:
We could maybe use the `//` to add some meaningful comment too, as it's a somewhat complex call.